lib/prometheus_exporter/server/active_record_collector.rb

```diff
@@ -0,0 +1,48 @@
+module PrometheusExporter::Server
+  class ActiveRecordCollector < TypeCollector
+    def type
+      "active_record"
+    end
+
+    def collect(obj)
+      default_labels = { query: obj["query"], action: obj["action"] }
+      custom_labels = obj["custom_labels"]
+      labels = custom_labels.nil? ? default_labels : default_labels.merge(custom_labels)
+
+      ensure_active_record_metrics
+      @active_record_query_duration_seconds.observe(obj["duration"], labels)
+      @active_record_query_duration_seconds_summary.observe(obj["duration"], labels)
```

Comment:
I'm thinking of removing this query from the summary metric. Probably keep the action label.

@lcasassa do you think it makes sense the add the action label to the summary metric?? The action label tells us the kind of query, for example `User Load`, `User Update`.